local/index.js

```diff
@@ -12,6 +12,37 @@ const globalStyles = {
   typography: {}
 };
 
+const createQueryParams = () => {
+  const params = {};
+  const searchParams = window.location.search;
+  if (searchParams) {
+    const urlParams = searchParams.replace('?', '').split('&');
+    urlParams.forEach(param => {
+      const [key, value] = param.split('=');
+      params[key] = decodeURI(value);
+    });
+  }
+  return params;
+};
+const canonicalUrl = queryParams => {
+  let searchString = '';
+  const queries = Object.keys(queryParams);
+  queries.forEach((query, index) => {
+    index === 0 ? (searchString += '?') : (searchString += '&');
+    searchString += `${query}=${queryParams[query]}`;
+  });
+  return window.location.origin + searchString;
+};
+
+const utils = {
+  ...window.ElementSdk,
+  css: aphrodite.css,
+  StyleSheet: aphrodite.StyleSheet,
```

Comment:
what about `classes`, shouldn't that also be provided as a prop/util to the block? Seems like it should be a prop based on this:
https://github.com/volusion/element-documentation/blob/master/how-to/style-a-block-with-aphrodite/README.md

Reply:
@dan-grayson classes is generated by the `createFactory` function that is imported and processed in `src/index.js`